BUILD.md

```diff
@@ -0,0 +1,39 @@
+Environment
+This documentation is intended for Ubuntu 20.4
+
+### Docker
+
+To begin with, you should install Docker. If you haven't installed it yet, please run the following command:
+
+`bash docker_install.sh`
+
+### Verifier
+
+Build the Docker image for the verifier:
+
+`docker build -t acs:0.1 verifier/
+`docker run -d -p 2323:2323 -it acs:0.1 bash /root/startup.sh >/dev/null 2>&1`
+
+
+### Backend
+
+Run the backend:
+
+`bash backend/backend.sh`
+
+If you see the following message, it means that attestation has been completed:
+
+`INFO: validateCertificate: AK certificate verified against the PCA root`
+
+
+### Frontend
+
+Run the frontend:
+
+`bash frontend/frontend.sh`
+
+If you see that the virtual machine has successfully started, it means that the frontend and backend can communicate with each other. You can also enter the virtual machine (root: 123456) and check if the TPM is detected by running the following command:
+
```

Comment:
Please note that the default user is root and the password is 123456

Reply:
Based on the discussion at https://github.com/inclavare-containers/confidential-vtpm/pull/4#discussion_r1186709950, I have already changed the method of configuring the password. it will be removed soon